tests/cmd/main_test.py

```diff
@@ -6,6 +6,7 @@
 import os.path
 
 import mock
+from six import StringIO
```

Comment:
and actually, this is just a test so it shouldn't have been added as a dependency in setup.py (setup.py is for runtime dependencies, whereas this is a test-time dependency (and therefore belongs in requirements-dev.txt))